lib/Analysis/BlockFrequencyInfoImpl.cpp

```diff
@@ -138,8 +138,9 @@ static void combineWeight(Weight &W, const Weight &OtherW) {
 static void combineWeightsBySorting(WeightList &Weights) {
   // Sort so edges to the same node are adjacent.
   std::sort(Weights.begin(), Weights.end(),
-            [](const Weight &L,
-               const Weight &R) { return L.TargetNode < R.TargetNode; });
+            [](const Weight &L, const Weight &R) {
```

Comment:
Remove changes unrelated to the main one. I suspect you ran clang-format but without --style LLVM.